client/beadledom-client-example/example-client/src/main/java/com/cerner/beadledom/client/example/client/ExampleOneClientModule.java

```diff
@@ -54,9 +54,9 @@ PaginatedClientResource providePaginatedResource(
   BeadledomClientConfiguration provideClientConfig() {
     return BeadledomClientConfiguration.builder()
         .maxPooledPerRouteSize(60)
-        .socketTimeoutMillis(60)
-        .connectionTimeoutMillis(60)
-        .ttlMillis(60)
+        .socketTimeoutMillis(200)
```

Comment:
I was getting read timeouts for integration tests in ClientServiceSpec so i bumped this up a little bit. Let me know if this is a big mistake